docs/blog/2020-06-16-Gatsby-slideshow-with-Posenet/index.md

```diff
@@ -330,27 +322,27 @@ And just like that we can now control slide decks remotely!
 
 ![A sample slide deck being presented and viewed in parallel](./images/remote-slides.gif)
 
-## Get the code:
+## Get The Code:
 
 [Personal Site Repo](https://github.com/slarsendisney/personal-site)
 
 [Socket Server](https://github.com/slarsendisney/sld-clicker)
 
 ## Extension: Gesture Controlling My Slides
 
-[I recently attended a conference called “Halfstack”](https://sld.codes/articles/My-learnings-from-HalfStack's-Conference). While I was there I saw a demo of [Posenet](https://www.npmjs.com/package/@tensorflow-models/posenet). An awesome library that uses a model to identify key body parts from an image or video. I thought it would be cool to use this library to create touch-free slide navigation for presenters using just their gestures.
+[I recently attended a conference called “HalfStack”](https://sld.codes/articles/My-learnings-from-HalfStack's-Conference). While I was there I saw a demo of [PoseNet](https://www.npmjs.com/package/@tensorflow-models/posenet). An awesome library that uses a model to identify key body parts from an image or video. I thought it would be cool to use this library to create touch-free slide navigation for presenters using just their gestures.
 
-### How Posenet Works
+### How PoseNet Works
 
-Posenet uses a trained model to take an image and make an attempt at guessing the location of key body parts in that photo. You can see how, in the preview below, the skeleton is very closely matching my actual position — I was super impressed with its accuracy.
+PoseNet uses a trained model to take an image and make an attempt at guessing the location of key body parts in that photo. You can see how, in the preview below, the skeleton is very closely matching my actual position — I was super impressed with its accuracy.
 
 ![Man gesturing with two hands to the right and left to move slides forward and backward respectively](./images/gesture-control.gif)
 
 Being a magician! 🎩
 
 It can be implemented relatively easily (code borrowed from [Kirsten Lindsmith's "PoseNet for React" repo on Github](https://github.com/kirstenlindsmith/PoseNet_React/blob/master/client/components/Camera.js)):
 
-```js
+```jsx
```

Comment:
i would like to have here a file name, because i don't know where it belongs to (i can not find this code snipped in the linked server and client repos)

````suggestion
```jsx:title=???
````

Reply:
As the line above mentions - code is borrowed from Kirsten Lindsmith and is not my own. File name here would not be appropriate.

Reply:
ah ok - i was thinking you have implemented it for your site

Reply:
I have - but here I have tried to keep the code blocks easier to understand than the mess my code became on my site haha :P

Reply:
ok - i mark this comment as closed

Reply:
mhh - ok but i did not found it in your repo 🤔

Reply:
Its a private repo for now :)

Reply:
ok